### DISCUSSION TITLE
fastapi-users for companies and organisations

### DISCUSSION THREAD

**Discussion by dineshbvadhia** [fastapi-users/fastapi-users#941] [Q&A]
Does a fastapi-users exist for companies and organisations?

**Comment by frankie567** [comment]
Hi @dineshbvadhia!

I'm sorry but I don't get what you mean by that? FastAPI Users is open-source under the MIT license, so people and companies are free to use it for any project.

**Reply by dineshbvadhia** [reply]
fastapi-users is for managing users? Is there an equivalent for managing organizations? Or, is the idea to adapt fastapi-users for orgs?

**Reply by frankie567** [reply]
Yes, it's for managing users (registration, login, password reset, etc.). If you need to manage organizations, you'll need to implement that yourself.
